doc/source/data/doc_code/torch_image_batch_trained.py

```diff
@@ -37,7 +37,7 @@ def __init__(self):  # <1>
         self.model = resnet18(pretrained=True).cuda()
         self.model.eval()
 
-    def __call__(self, batch: List[torch.Tensor]):  # <2>
+    def __call__(self, batch: Dict[str, np.ndarray]):  # <2>
         torch_batch = torch.stack(batch).cuda()  # <3>
```

Comment:
don't we need to change this to `torch.stack(batch["images"].tolist())`? stack doesn't work with a dictionary

Reply:
Yeah I changed this to:

```
    def __call__(self, batch: Dict[str, np.ndarray]) -> Dict[str, np.ndarray]:  # <2>
        torch_batch = torch.stack(batch["preprocessed"]).cuda()  # <3>
```

In here already: https://github.com/ray-project/ray/pull/34880

Waiting to see if tests pass.